bundle.proto

```diff
@@ -14,3 +14,55 @@ message BundleUuid {
   bundle.Bundle bundle = 1;
   string uuid = 2;
 }
+
+/* Bundle Result Types */
+
+// Indicates the bundle was forwarded to a validator.
+// NOTE: A single bundle may have multiple events emitted if forwarded to many validators.
+message Forwarded {
+  // Slot at which bundle was forwarded.
+  uint64 slot = 1;
+
+  // Validator identity bundle was forwarded to.
+  string validator_identity = 2;
+}
+
+// Indicates the bundle was dropped and therefore not forwarded to any validator.
+message Dropped {
+  oneof reason {
+    AuctionLost auction_lost = 1;
+    SimulationFailure simulation_failure = 2;
+    InternalError internal_error = 3;
+  }
+}
+
+// Indicates the bundle lost the auction.
+message AuctionLost {
+  // Auction's unique identifier.
+  string auction_id = 1;
+
+  // Bundle's simulated bid.
+  uint64 simulated_bid_lamports = 2;
+}
+
+// Bundle dropped due to simulation failure.
+message SimulationFailure {
+  // Signature of the offending transaction.
+  string tx_signature = 1;
+  optional string msg = 2;
+}
+
+// Bundle dropped due to an internal error.
+message InternalError {
+  string msg = 1;
+}
+
+message BundleResult {
+  // Bundle's Uuid.
+  string bundle_id = 1;
+
+  oneof result {
```

Comment:
should we make this its own type?

Reply:
we can but that would add one more level of indirection:

e.g.
```
bundle_result.result.result.thing
```
vs.
```
bundle_result.result.thing
```

what's the reason for wrapper type?